.github/workflows/README.md

```diff
@@ -35,27 +68,27 @@ The CD Workflow is responsible for automatically deploying new code changes to d
 
 
 
-**Workflow Trigger and Jobs**
-
-The CD Workflow is triggered by manual trigger through workflow dispatch, allowing developers to initiate the deployment process on demand.
-
-The CD Workflow consists of several jobs, each responsible for a specific deployment task. For instance, the `deploy-to-dev` job deploys the dev build to Firebase, while the `promote-artifact` job promotes the release build to the specified Google Play track.
-
-## **Dependency Updates workflow**
+**Version Code**
 
-Updates project dependencies using Dependabot, an automated dependency management tool, ensuring that the project always uses the latest stable versions of its dependencies.
+The version code is generated from 3 things:
+1. A "base" version code of `10000000`. For a time the version code was derived from unix time / 1000. This created a always increasing number, but was otherwise not so useful. Setting a base allows us to establish a clean "floor" for the version code. If the workflow runs ever exceed the base, first of all go us, and 2nd the base can be removed. 
```

Comment:
Ah, you went with 10000000 instead of 9000000. That's longer but kind of "cleaner".